test/publish.sh

```diff
@@ -74,7 +74,9 @@ EOF
 )
 
 # "Temporary" hacks
-export PATH=${GOPATH}/bin:${PATH}
+if [[ -z ${GOPATH} ]]; then
```

Comment:
because of `set -eux` this will still fail if gopath is not set


I highly recommend you run this in a docker container so you build with the same environment as we do in official releases

Reply:
OK. I did it run it in a docker container yesterday,  or did you mean I should run this with gcr.io/istio-testing/build-tools?